Marlin/src/feature/pause.cpp

```diff
@@ -252,42 +260,67 @@ void wait_for_filament_reload(const int8_t max_beep_count/*=0*/) {
 }
 
 void resume_print(const float &load_length/*=0*/, const float &initial_extrude_length/*=0*/, const int8_t max_beep_count/*=0*/) {
-  bool nozzle_timed_out = false;
-
   if (!move_away_flag) return;
-
+  bool nozzle_timed_out = false;
+  bool swap_spool_disabled = true;
+  int16_t fansp=fanSpeeds[ADVANCED_PAUSE_FAN];
```

Comment:
swap_spool_disabled , is an addition because , i need to inhibit all menus when swapping spool , i have created this data because when ' spool swapping is active ' all menus are deactivated , 
I don't have any other idea , the only idea i had is to make two big block of code , one with ' spool swap ' and the other ' orginal ' , but too loud when reading
I don't know what you make in this situation